mappings/net/minecraft/client/texture/atlas/source/UnstitchSpriteSource.mapping

```diff
@@ -0,0 +1,34 @@
+CLASS net/minecraft/unmapped/C_kafxtemf net/minecraft/client/texture/atlas/source/UnstitchSpriteSource
+	FIELD f_bfakdnfd TEXTURE_ID_NAMESPACE Lnet/minecraft/unmapped/C_iiomeaxy;
+	FIELD f_dtciriyo id Lnet/minecraft/unmapped/C_ncpywfca;
+	METHOD <init> (Lnet/minecraft/unmapped/C_ncpywfca;Ljava/util/List;DD)V
+		ARG 1 id
+		ARG 2 regions
+		ARG 3 divisorX
+		ARG 5 divisorY
+	METHOD m_lcdtbyjj (Lcom/mojang/serialization/codecs/RecordCodecBuilder$Instance;)Lcom/mojang/datafixers/kinds/App;
+		ARG 0 instance
+	CLASS C_akjjowfy LazyLoadedImage
+		FIELD f_begbulaa id Lnet/minecraft/unmapped/C_ncpywfca;
+		FIELD f_iqefkrmq resource Lnet/minecraft/unmapped/C_tuynegco;
+		FIELD f_ivbuxcxm referenceCount Ljava/util/concurrent/atomic/AtomicInteger;
+		FIELD f_ldwbrifm image Ljava/util/concurrent/atomic/AtomicReference;
+		METHOD <init> (Lnet/minecraft/unmapped/C_ncpywfca;Lnet/minecraft/unmapped/C_tuynegco;I)V
+			ARG 1 id
+			ARG 2 resource
+			ARG 3 regionCount
+		METHOD m_izmxfzcg release ()V
+		METHOD m_kpiuvtsr get ()Lnet/minecraft/unmapped/C_ayikuhxa;
+	CLASS C_arxmarph RegionGenerator
+		FIELD f_bhzsadxn image Lnet/minecraft/unmapped/C_kafxtemf$C_akjjowfy;
+		FIELD f_bibpunia region Lnet/minecraft/unmapped/C_kafxtemf$C_wpyfoogz;
+		FIELD f_gpnbyfhy yDivisor D
```

Comment:
Why are these `x/yDivisor` but lines 7/8 are `divisorX/Y`?

Reply:
ah this is an oversight, thanks, i thought something was different when mapping